### PR TITLE
Perform major surgery on CGContext.

### PR DESCRIPTION
**Pull request by DHowett-MSFT**
**This pull request contains no functional changes.**

This is part 2 of the CGContext cleanup; I've categorized every method and moved them into reasonable regions. We had two copies of `CGContextShowGlyphsWithAdvances`, one of which has been removed.

Verification method: 
```
$ diff -u <(git show 29e1b23:./CGContext.mm|sort) <(sort CGContext.mm | dos2unix)
...
```

Sorting a source file to diff it is silly, but it shows that the bulk of the changes were in region renames and the deletion of CGContextShowGlyphsWithAdvances.

### CODE REVIEW

File: Frameworks/CoreGraphics/CGContext.mm [hunk 1]
```diff
@@ -419,891 +444,969 @@ void CGContextSetCTM(CGContextRef context, CGAffineTransform transform) {
...
+/**
+ @Status Stub
+ @Notes
+*/
+bool CGContextPathContainsPoint(CGContextRef context, CGPoint point, CGPathDrawingMode mode) {
+    NOISY_RETURN_IF_NULL(context, StubReturn());
+    UNIMPLEMENTED();
+    return StubReturn();
 }
+#pragma endregion
```
Comment (rajsesh-msft) [inline Frameworks/CoreGraphics/CGContext.mm]: @DHowett-MSFT other than arranging the file to be more logical, are there real code changes that need to be reviewed?

**Comment by ms-jihua** [comment]
:shipit:

File: Frameworks/CoreGraphics/CGContext.mm (hunk 1 above)
Comment (rajsesh-msft) [inline Frameworks/CoreGraphics/CGContext.mm]: I approve this pull request.

**Comment by msft-Jeyaram** [comment]
:shipit:

**Pull request merged**